src/displayapp/screens/WatchFaceAnalog.cpp

```diff
@@ -15,8 +15,10 @@ namespace {
   constexpr int16_t MinuteLength = 90;
   constexpr int16_t SecondLength = 110;
 
+  // LVGL sine isn't constexpr (thought it could be) so fix size here
```

Comment:
typo?
```suggestion
  // LVGL sin isn't constexpr (thought it could be) so fix size here
```

Reply:
It's the full spelling https://en.wikipedia.org/wiki/Sine_and_cosine
I think it's probably clearer your way